### PR TITLE
ADK version support for Server OS deployment

### PR DESCRIPTION
**Pull request by josemrodrigues**
Could we update the article with which ADK version is supported for Server OS deployment for each server OS version

### CODE REVIEW

**Comment by PRMerger17** [comment]
@josemrodrigues : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/core/plan-design/configs/support-for-windows-10.md [hunk 1]
```diff
@@ -79,3 +79,6 @@ The following table lists the versions of the Windows 10 ADK that you can use wi
 
  > [!Note]  
  > Configuration Manager only supports x86 and amd64 components of the Windows 10 ADK. It doesn't currently support ARM or ARM64 components. 
+
+# ADK version support for Server OS deployment:
```
Comment (aczechowski) [inline sccm/core/plan-design/configs/support-for-windows-10.md]: Please change from H1 header to a tip, for example: 
`> [!Tip]`

File: sccm/core/plan-design/configs/support-for-windows-10.md [hunk 2]
```diff
@@ -79,3 +79,6 @@ The following table lists the versions of the Windows 10 ADK that you can use wi
 
  > [!Note]  
  > Configuration Manager only supports x86 and amd64 components of the Windows 10 ADK. It doesn't currently support ARM or ARM64 components. 
+
+# ADK version support for Server OS deployment: 
+Treat Server 2016 like Windows 10 LTSB 2016 as both of them are 10.14393.x
```
Comment (aczechowski) [inline sccm/core/plan-design/configs/support-for-windows-10.md]: Please tweak the wording on this, something like, 
`Windows Server builds have the same ADK requirement as the associated Windows 10 version. For example, Windows Server 2016 is the same build version as Windows 10 LTSB 2016.`
I think something simple like that is sufficient. What do you think?

**Comment by PRMerger13** [comment]
@josemrodrigues : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/core/plan-design/configs/support-for-windows-10.md (hunk 1 above)
Comment (josemrodrigues) [reply sccm/core/plan-design/configs/support-for-windows-10.md]: title changed

File: sccm/core/plan-design/configs/support-for-windows-10.md (hunk 2 above)
Comment (josemrodrigues) [reply sccm/core/plan-design/configs/support-for-windows-10.md]: I agree with that thanks

**Comment by aczechowski** [comment]
One more formatting tweak, add an angle bracket `>` to the start of line 85 to group it in the tip. For example:

`> Windows Server...`

**Comment by PRMerger15** [comment]
@josemrodrigues : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @josemrodrigues for the quick responses. 

*Once merged*, this should be live with the next daily publishing process. Right now that *could* be later today around 11:00 Pacific time. Otherwise, tomorrow (Thursday, 23 August) about the same time.

#sign-off

**Pull request merged**